src/cider/exec/nextgen/context/CodegenContext.h

```diff
@@ -92,6 +125,16 @@ class CodegenContext {
         : ctx_id(id), name(n), capacity(c), initializer_(initializer) {}
```

Comment:
declare a default virtual destructor here, or there will be a new-delete-mismatch when using ```AggBufferDescriptor```, you could enable ASN and test this problem.